python/tvm/relay/frontend/onnx.py

```diff
@@ -1482,6 +1482,20 @@ def _impl_v12(cls, inputs, attr, params):
             result = _op.expand_dims(result, axis)
         return result
 
+class Squeeze(OnnxOpConverter):
+    """Operator converter for Squeeze."""
+
+    @classmethod
+    def _impl_v1(cls, inputs, attr, params):
+        return _op.squeeze(*inputs, axis=attr["axes"])
```

Comment:
You must handle the case where "axes" is not given: https://github.com/onnx/onnx/blob/master/docs/Changelog.md#Squeeze-1